core-android/build.gradle

```diff
@@ -41,8 +41,8 @@ android {
     }
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_7
-        targetCompatibility JavaVersion.VERSION_1_7
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
```

Comment:
will this force the 3p now to be on this version? there maybe some clients who are still on 1.7?

Reply:
yes, we already upgraded our java-sdk to 1.8 so we need this change